### PR TITLE
Minor doc markup fixes.

### PR DESCRIPTION
**Pull request by anntzer**
This should finish closing #7205.

Also note the funny typo previously in the docs of the `minor` kwarg...

## PR Checklist

- [NA] Has Pytest style unit tests
- [NA] Code is PEP 8 compliant 
- [NA] New features are documented, with examples if plot related
- [X] Documentation is sphinx and numpydoc compliant
- [NA] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [NA] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

**Comment by tacaswell** [comment]
rebase and force-pushed to pick up corrected circle config

File: lib/matplotlib/ticker.py [hunk 1]
```diff
@@ -38,99 +38,94 @@
...
-    locator for index plots (e.g., where x = range(len(y)))
+    Locator for index plots (e.g., where ``x = range(len(y))``).
 
 :class:`LinearLocator`
-    evenly spaced ticks from min to max
+    Evenly spaced ticks from min to max.
 
 :class:`LogLocator`
-    logarithmically ticks from min to max
+    Logarithmically ticks from min to max.
```
Comment (dopplershift) [inline lib/matplotlib/ticker.py]: "Logarithmically spaced"?

File: lib/matplotlib/ticker.py [hunk 2]
```diff
@@ -38,99 +38,94 @@
...
+    part outside of the threshold and add 0 if inside the limits.
 
 :class:`MultipleLocator`
-    ticks and range are a multiple of base; either integer or float
+    Ticks and range are a multiple of base; either integer or float.
 
 :class:`OldAutoLocator`
-    choose a MultipleLocator and dyamically reassign it for
-    intelligent ticking during navigation
+    Choose a `MultipleLocator` and dyamically reassign it for intelligent
```
Comment (dopplershift) [inline lib/matplotlib/ticker.py]: "dyamically" -> "dynamically".

**Comment by tacaswell** [comment]
fyi @anntzer I force pushed to your branch a couple of times

**Comment by dstansby** [comment]
Needs a rebase, but looks good to me

**Comment by anntzer** [comment]
done

**Pull request merged**